ntbs-service/Migrations/20210422154810_UserTableUsesIdAsPrimaryAndForeignKeys.cs

```diff
@@ -0,0 +1,324 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using ntbs_service.Models.Entities;
+
+namespace ntbs_service.Migrations
+{
+    public partial class UserTableUsesIdAsPrimaryAndForeignKeys : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Drop foreign and primary keys and indexes
+            migrationBuilder.DropForeignKey(
+                name: "FK_Alert_User_CaseManagerUsername",
+                table: "Alert");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_CaseManagerTbService_User_CaseManagerUsername",
+                table: "CaseManagerTbService");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_HospitalDetails_User_CaseManagerUsername",
+                table: "HospitalDetails");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_TreatmentEvent_User_CaseManagerUsername",
+                table: "TreatmentEvent");
+
+            migrationBuilder.DropPrimaryKey(
+                name: "PK_CaseManager",
+                table: "User");
+
+            migrationBuilder.DropIndex(
+                name: "IX_TreatmentEvent_CaseManagerUsername",
+                table: "TreatmentEvent");
+
+            migrationBuilder.DropIndex(
+                name: "IX_HospitalDetails_CaseManagerUsername",
+                table: "HospitalDetails");
+
+            migrationBuilder.DropPrimaryKey(
+                name: "PK_CaseManagerTbService",
+                table: "CaseManagerTbService");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Alert_CaseManagerUsername",
+                table: "Alert");
+            
+            // Add new columns
+            migrationBuilder.AddColumn<int>(
+                    name: "Id",
+                    table: "User",
+                    type: "int",
+                    nullable: false,
+                    defaultValue: 0)
+                .Annotation("SqlServer:Identity", "1, 1");
+
+            migrationBuilder.AddColumn<int>(
+                name: "CaseManagerId",
+                table: "TreatmentEvent",
+                type: "int",
+                nullable: true,
+                defaultValue: 0);
```

Comment:
Surely we shouldn't have a default here (because omitting it would leave a NULL ID, which is what we want)?

Reply:
Yes I'm not sure why these default values showed up, I will remove it